nservicebus/messaging/messages-events-commands.md

```diff
@@ -20,72 +20,65 @@ redirects:
  - nservicebus/messaging/invalidoperationexception-in-unobtrusive-mode
 ---
 
-A *Message* is the unit of communication for NServiceBus. There are two sub-types of messages that capture more of the intent and help NServiceBus enforce messaging best practices. This enforcement is enabled by default unless disabled in [configuration](best-practice-enforcement.md).
+A *message* is the unit of communication for NServiceBus. There are two types of messages, _commands_ and _events_, that capture more of the intent and help NServiceBus enforce messaging best practices. This enforcement is enabled by default, but can be [disabled](best-practice-enforcement.md).
 
-Note: Messages must be classes. Using structs will generate exceptions at runtime when attempting to send.
+Note: A message must be a reference type (`class`). An attempt to send a message which is a value type (`struct`) will result in a runtime exception.
 
+Command | Event
+-- | --
+Used to _make a request to perform an action_. | Used to _communicate that an action has been performed_.
+Has one logical owner. | Has one logical owner.
+Should be _sent to_ the logical owner. | Should be _published by_ the logical owner.
+Cannot be _published_. | Cannot be _sent_.
+_Cannot_ be subscribed to or unsubscribed   from. | _Can_ be subscribed to and unsubscribed from.
+_Can_ be sent using the [gateway](/nservicebus/gateway). | _Cannot_ be sent using the [gateway](/nservicebus/gateway).
 
-### Command
+Note: In a request and response pattern, _reply_ messages are neither a command nor an event.
 
-Used to request that an action should be taken. A *Command* is intended to be _sent to a receiver_ (all commands should have one logical owner and should be sent to the endpoint responsible for processing). As such, commands:
 
- * Are not allowed to be _published_.
- * Cannot be _subscribed_ to or _unsubscribed_ from.
- * Cannot implement `IEvent`.
+### Validation
 
+There are checks in place to ensure best practices are followed. Violations of the above guidelines generate the following exceptions:
 
-### Event
-
-Used to communicate that some action has taken place. An *Event* should be _published_. An event:
-
- * Can be _subscribed_ to and _unsubscribed_ from.
- * Cannot be sent using `Send()` (since _all events should be published_).
- * Cannot implement `ICommand`.
- * Cannot be sent using the gateway, i.e. `SendToSites()`.
+partial: errors
 
-Note: For reply messages in a request and response pattern, use `IMessage` since these replies are neither a Command nor an Event.
 
+ ## Designing messages
 
-### Validation Messages
+ Messages represent data contracts between endpoints. They should be designed according to the following guidelines.
+ 
+ Messages should:
 
-There are checks in place to ensure following of the best practices. While violating above rules the following exceptions can be seen:
+ * be simple [POCO](https://en.wikipedia.org/wiki/Plain_old_CLR_object) objects.
```

Comment:
Having _objects_ at the end there is redundant but admittedly reads better

Reply:
Yeah, true. That got carried over from the original text which I moved from below. How about "POCO types"?